### ISSUE TITLE
The Reproduction Report

### ISSUE THREAD

**Issue by shyanguan** [mkocabas/VIBE#98]
Here is the puzzle I met during my experiments. List them out for discussion.

1.  I have many different points with @ZephirGe's training log in this [issue](https://github.com/mkocabas/VIBE/issues/69). Details:
 * In the beginning, the performance is already 54/ 55. This is really confusing since the reported SPIN in 3DPW is 59. Consequently, someone could check this again?
* I found the size of his evaluation dataset is 67, while mine is 20. So how about you?

2. I tried to add human 3.6M into the training set, but I didn't modify other configs, such as 2D dataset ratio etc.. But in this case, surprisingly , the performance slightly decrease. Anyone can explain this phenomenon?

3. Last but most important, I still cannot reproduce the results reported in the paper. Hope the author can release more hints, such as their training log for comparing.

**Comment by shyanguan** [comment]
Also, I think if authors claim the h36, Kinetics-400, Penn action is useless, they should attach some convincing results.

**Comment by mkocabas** [comment]
Hi @shyanguan,

Thanks for your analysis. Here is my answers to your questions:

1. Training converges fast and this is expected since temporal information has better clues than single frame methods. Dataset size difference might be related to the missing data. I would suggest you to compare the data you preprocessed with the official releases of each dataset.

2. This is normal, and happens with other methods as well. Because, Human3.6M is an indoor dataset with limited numbers of subject. Training on it does not contribute to the in-the-wild performance ie. 3DPW accuracy in our experiments, too. Also, I would suggest you to use the visualization functions we have provided to check whether the preprocessed data is correct.

3. There is no particular hints except the instructions we documented. I will attach the training logs I have as soon as possible.
